router/router.go

```diff
@@ -27,4 +41,72 @@ func SetupRoutes(app *fiber.App) {
 	user.Post("/", handler.CreateUser)
 	user.Patch("/:id", middleware.Protected(), handler.UpdateUser)
 	user.Delete("/:id", middleware.Protected(), handler.DeleteUser)
+
+	app.Get("/", func(c *fiber.Ctx) error {
```

Comment:
I think it would make sense to move the function body into handler (like it's done for endpoints above) to keep router clean